### PR TITLE
fix: default pool device to aio if not specified

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Requirement coming from FM-40
fixes CAS-848

### CODE REVIEW

**Review by jonathan-teh — APPROVED** [review]
Looks fine, presumably this address CAS-848.

**Comment by tiagolobocastro** [comment]
> Looks fine, presumably this address CAS-848.

yes that's the one

File: mayastor/src/lvs/lvs_pool.rs [hunk 1]
```diff
@@ -306,15 +306,7 @@ impl Lvs {
...
-                    format!(
-                        "{}://{}",
-                        if uring::kernel_support() {
-                            "uring"
-                        } else {
-                            "aio"
-                        },
-                        d,
-                    )
+                    format!("aio://{}", d)
```
Comment (paulyoong) [inline mayastor/src/lvs/lvs_pool.rs:309]: Update comment on line 303 as functionality has changed.

**Review by paulyoong — APPROVED** [review]

File: mayastor/src/lvs/lvs_pool.rs (hunk 1 above)
Comment (tiagolobocastro) [reply mayastor/src/lvs/lvs_pool.rs:309]: thanks, done

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**